assets/js/modules/search-console/components/dashboard/SearchFunnelWidget/Overview.js

```diff
@@ -158,33 +174,49 @@ const Overview = ( {
 	return (
 		<Grid>
 			<Row>
-				<Cell { ...quarterCellProps }>
-					<DataBlock
-						stat={ 0 }
-						className="googlesitekit-data-block--impressions googlesitekit-data-block--button-1"
-						title={ __( 'Total Impressions', 'google-site-kit' ) }
-						datapoint={ totalImpressions }
-						change={ totalImpressionsChange }
-						changeDataUnit="%"
-						context="button"
-						selected={ selectedStats === 0 }
-						handleStatSelection={ handleStatsSelection }
-					/>
-				</Cell>
+				{ isSearchConsoleGatheringData && ! hasSearchConsoleData && (
+					<Cell { ...halfCellProps }>
+						<WidgetReportZero moduleSlug="search-console" />
+					</Cell>
+				) }
+
+				{ ! isSearchConsoleGatheringData && hasSearchConsoleData && (
```

Comment:
The `DataBlock` below should be shown so long as it is not gathering data, even if it is  zero data. It's a bit confusingly named perhaps, but gathering data == WidgetReportZero